mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -374,12 +373,34 @@ def evaluate(self,
                 else:
                     gt_file = osp.join(self.img_prefix, f'{name}/gt/gt.txt')
                 res_file = osp.join(resfiles['track'], f'{name}.txt')
-                gt = mm.io.loadtxt(gt_file)
+                # copy gt file from ceph to local temporary directory
+                # no 'self.img_prefix'
+                gt_dir_path = osp.join(local_dir.name, name, 'gt')
+                os.makedirs(gt_dir_path)
+                gt_file_tmp = local_dir.name + gt_file.replace(
```

Comment:
Using `+` may be necessary, the `local_dir.name` is `/tmp/*******` and the `gt_file.replace` is `/mot_name/gt/gt.txt`. Using `osp.join` may be a dangerous operation, and `os.sep` should also be unnecessary. I wonder if you have some suggestions?

Reply:
I still don't get why we can't use osp.join?